webdriver/tests/bidi/browsing_context/close/prompt_unload.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+import asyncio
+
+from webdriver.bidi.modules.input import Actions
+
+pytestmark = pytest.mark.asyncio
+
+CONTEXT_DESTROYED_EVENT = "browsingContext.contextDestroyed"
+USER_PROMPT_OPENED_EVENT = "browsingContext.userPromptOpened"
+
+
+@pytest.mark.parametrize("prompt_unload", [None, False])
+async def test_prompt_unload_not_triggering_dialog(bidi_session, inline, subscribe_events, top_context, prompt_unload):
+    url = inline(
+        "<script>window.addEventListener('beforeunload', event => {event.preventDefault();});</script>")
+
+    # Set up event listener to make sure the event is not automatically handle
```

Comment:
Where in BiDi do we specify that the prompt is automatically handled if no event listener is subscribed? I think that in this case it should be more the following?

` Set up event listener to make sure the "beforeunload" event is not emitted`